Content/package.json

```diff
@@ -10,8 +10,8 @@
         "copy-webpack-plugin": "^4.6.0",
```

Comment:
can we get rid of explicit reference of core-js? @alfonsogarciacaro

Reply:
https://github.com/fable-compiler/webpack-config-template/issues/12#issuecomment-490862309

Reply:
Yes, unfortunately it seems that now you need to make the core-js version explicit **both** in package.json and webpack.config.json :/